lib/osf-components/addon/components/registries/schema-block-renderer/read-only/multi-select/template.hbs

```diff
@@ -27,9 +27,11 @@
             {{t 'osf-components.registries.schema-block-renderer/read-only/response.noResponse'}}
         </p>
     {{/if}}
-    <ValidationErrors
-        local-class='ValidationErrors'
-        @changeset={{@changeset}}
-        @key={{@schemaBlock.registrationResponseKey}}
-    />
+    {{#if @changeset}}
```

Comment:
This change feels out of place.

Reply:
we are now using these components both on the overview and submission pages. On the overview, we're leaving empty error divs.